src/container.c

```diff
@@ -224,6 +224,55 @@ static int container_setup_mount(struct hyper_container *container)
 	return 0;
 }
 
+static int container_recreate_file(char *filename)
+{
+	fprintf(stdout, "recreate file %s\n", filename);
+	hyper_unlink(filename);
+	return hyper_create(filename);
+}
+
```

Comment:
hyperstart already create mountpoint when it need to mount something.

but container_recreate_file() might be still need.

current hyperstart create the mountpoints without unlink them at first.
it will have problems when the user intently create the mountpoint in they image at first.
1) file type dismatch (fail to mount)
2) file permission dismatch (user process might fail)

but these hardly happen (when the user intently create the wrong
mountpoint in they image at first.) But IMO, it is the user's responsibility for the fail.
and in most cases, the container doesn't touch the rootfs(`hyper run ubuntu ls`),
and the `unlink` add burden to the storage driver (especially rbd driver).

I think we can check the files, if the file type & mode match we don't unlink them?